waku/v2/waku_rln_relay/rln/wrappers.nim

```diff
@@ -51,30 +53,69 @@ proc membershipKeyGen*(ctxPtr: ptr RLN): RlnRelayResult[IdentityCredential] =
 
   return ok(identityCredential)
 
-proc createRLNInstanceLocal*(d: int = MerkleTreeDepth): RLNResult =
+type RlnTreeConfig* = ref object of RootObj
+  cache_capacity*: int
+  mode*: string
+  compression*: bool
+  flush_interval*: int
+  path*: string
+
+type RlnConfig* = ref object of RootObj
+  resources_folder*: string
+  tree_config*: RlnTreeConfig
+
+proc `%`(c: RlnConfig): JsonNode =
+  ## wrapper around the generic JObject constructor.
+  ## We don't need to have a separate proc for the tree_config field
+  let tree_config = %{ "cache_capacity": %c.tree_config.cache_capacity,
+                       "mode": %c.tree_config.mode,
```

Comment:
Maybe very basic question but what does `%` do by default in Nim? :)

Reply:
ah true, I forgot it. Thanks! We do the same in https://github.com/waku-org/nwaku/blob/ba8ec7048e71f56e787a1e6afe86d7985b1bf7c2/library/events/json_message_event.nim#L42